src/game/camera.rs

```diff
@@ -106,7 +119,8 @@ impl Camera {
                               fov: Option<f32>,
                               aspect_ratio: Option<f32>,
                               near: Option<f32>,
-                              far: Option<f32>) -> &mut Camera {
+                              far: Option<f32>)
+                              -> &mut Camera {
```

Comment:
I find it a bit odd that the -> is aligned with the args rather than with '(', given that it's _outside_ the parens. But it's definitely fine.